### PR TITLE
mgr/dashboard: fix M retention frequency display

### PR DESCRIPTION
**Pull request by ivoalmeida**
Fix snap schedule list monthly retention frequency display

Fixes: https://tracker.ceph.com/issues/64982

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [x] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

image

### CODE REVIEW

**Comment by cloudbehl** [comment]
Its getting undefined. 
image

**Review by Pegonzal — APPROVED** [review]

File: src/pybind/mgr/dashboard/frontend/src/app/shared/enum/retention-frequency.enum.ts [hunk 1]
```diff
@@ -11,7 +12,8 @@ export enum RetentionFrequencyCopy {
   h = 'Hourly',
   d = 'Daily',
   w = 'Weekly',
-  m = 'Monthly',
+  M = 'Monthly',
+  m = 'Minutely',
```
Comment (cloudbehl) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/enum/retention-frequency.enum.ts:16]: Is it going to be in this order in the drop down? I see we are following `ascending order` for all items and according to that this should be first item.

Comment (ivoalmeida) [reply src/pybind/mgr/dashboard/frontend/src/app/shared/enum/retention-frequency.enum.ts:16]: good catch. done 👍🏽

**Review by cloudbehl — APPROVED** [review]
NIT: Can you update the commit message to reflect both Monthly fix and adding of Minutely item.

**Pull request merged**